src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int PRIME_NUMBER = 17;
+    private static final int MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size = 0;
+    private int threshold;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        threshold = (int) (DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        putVal(hash(key), key, value);
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        int index = hash(key) % table.length;
+        Node<K, V> currentNode = table[index];
+        while (true) {
+            if (currentNode == null) {
+                return null;
+            } else if (isKeyEquals(currentNode.key, key)) {
+                return currentNode.value;
+            }
```

Comment:
do we need else clause after return statement?

Reply:
removed